src/util/factory.ts

```diff
@@ -22,7 +22,7 @@ import ts from "typescript";
 export namespace f {
 	let factory = ts.factory;
 
-	type NodeArray<T extends ts.Node> = Array<T> | ts.NodeArray<T>;
+	type NodeArray<T extends ts.Node> = ReadonlyArray<T> | ts.NodeArray<T>;
```

Comment:
This was necessary to get `ts.getModifiers()` to play nicely in a few default arguments.

It doesn't seem to break anything and I think it's more accurate since `ts.NodeArray` extends from `ReadonlyArray`.